coding-standards/hooks/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+rootdir=$(git rev-parse --show-toplevel)
+
+cd $rootdir/coding-standards
+
+if ! pipenv run flake8 --help >>/dev/null 2>&1; then
+	pipenv sync
+fi
+
+if ! pipenv run black --check --exclude="/(\.git|build|dist|debian)/" --line-length=120 .. >>/dev/null 2>&1; then
+	echo "Please format your files with black before commiting"
+        echo "  pipenv run black --exclude='/(\.git|build|dist|debian)/' --line-length=120 $rootdir"
```

Comment:
nem kell elotte "cd coding-standards" ? honnet tudja hol kell futtatni?

Reply:
We put $rootdir at the end of the command, which tells black on which directory it sould run.
This way the user does not have to cd.

Reply:
Hmm, and how would pipenv know which environment to use ?

Reply:
Yeah you are right, we have to cd there

Reply:
Done. I created a script that handles the formatting